NoSoft-SA/nspack
# Feature/manual inspection passed flag for govt pallets

Summary of this change:
Added flag `manual_inspection_passed` to govt_inspection_pallets. This flag saves the same value as `govt_inspection_pallets.passed`.

However, if a Titan request were to override the value of `govt_inspection_pallets.passed`, its value is moved to `manual_inspection_passed`.

How to test this code via the UI:
See column values in govt_inspection_pallets grid, stock pallets grid and finished pallets grid.
Add a pallet to a govt sheet and check if the value equals the inspection result.
Check if when titan inspection overwrites passed, that the previous inspection result in saved on manual_inspection_passed
